app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ActiveRecord::Base
+  belongs_to :users, class_name: 'User', optional: true
+  belongs_to :posts, class_name: 'Post', optional: true
+
+  def update_like_counter
+    counter = Like.where(post_id: post_id).count
+    Post.update(post_id, likes_counter: counter)
```

Comment:
[REQUIRED] It is quicker and precise if we use the `increment!`. Kindly refer to the expression below
```ruby
post.increment!(:likes_counter)
```